packages/web-twig/src/Resources/components/TextFieldBase/README.md

```diff
@@ -71,7 +71,7 @@ Without lexer:
 | `name`              | `string`                                                      | `null`  | no       | Input name                                                              |
 | `placeholder`       | `string`                                                      | `null`  | no       | Input placeholder                                                       |
 | `type`              | `email`, `number`, `password`, `search`, `tel`, `text`, `url` | `text`  | no       | Input type                                                              |
-| `validationState`   | `success`, `warning`, `error`                                 | `null`  | no       | Type of validation state                                                |
+| `validationState`   | `success`, `warning`, `danger`, `error`                       | `null`  | no       | Type of validation state                                                |
```

Comment:
There should be link to docs/dictionaries.md and deprecation info as well.

Reply:
done